rlImGui/rlImGui.cpp

```diff
@@ -302,6 +334,8 @@ void SetupRLImGui(bool dark)
     ImGuiIO& io = ImGui::GetIO();
     io.BackendPlatformName = "imgui_impl_raylib";
 
+	io.BackendFlags |= ImGuiBackendFlags_HasMouseCursors;
```

Comment:
extra space.

Reply:
what do you mean exactly?

Reply:
if you are talking about the indenting thats fixed now